pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>es.tid.fiware</groupId>
     <artifactId>keypass</artifactId>
-    <version>1.2.1</version>
+    <version>1.3.0</version>
```

Comment:
IMHO more over that his PR there is not other change or feature to incorporate to this new release (version 1.3.0)